pcsx2/System.cpp

```diff
@@ -358,6 +358,27 @@ namespace HostMemoryMap {
 
 /// Attempts to find a spot near static variables for the main memory
 static VirtualMemoryManagerPtr makeMainMemoryManager() {
+	const wxString memoryManagerName = "Main Memory Manager";
+	
+	auto tryMakeAt = [&memoryManagerName](uptr base) -> VirtualMemoryManagerPtr {
+		auto mgr = std::make_shared<VirtualMemoryManager>(memoryManagerName, base, HostMemoryMap::Size, /*upper_bounds=*/0, /*strict=*/true);
+		if (mgr->IsOk())
+			return mgr;
+		return nullptr;
+	};
+
+	// Historically, the base address has always been 0x20000000 for x86 builds. Many cheat tables and third party
+	// tools relies to this specific address, therefore try to initialize the memory manager using this base
+	// address first. This does not guarantee it, but it prioritise the legacy memory address.
+	// This is only valid for x86 builds, as x64 have a high chance of failing an assertion by using this
+	// specific address.
+	//if (sizeof(void*) == 4)
```

Comment:
You accidentally left the `//` in

Reply:
That was definitively a distraction. Thanks for the flawless code review.